### DISCUSSION TITLE
locked account.. now i cant log in on other accounts anymore

### DISCUSSION THREAD

**Discussion by PatschD** [vercel/community#3875] [Account Issue] [answered]
### Question

Hey, i use vercel at work for some projects. I wanted to create a personal account for personal stuff but when i signed up with github i got the error the account was blocked. Then i tried to add the github account to my work account, but this caused the default account for login to switch to my personal github (which is blocked..). So now, when i try to sign in it automatically refers to my personal account.

why is my personal ccount blocked in the first place? i just signed up.

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble with login. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

To open a ticket please use the appropriate direct channel found in this guide: [What is the best way to get support from Vercel?](https://vercel.com/guides/how-to-get-vercel-support#direct-channels)
